packer-build/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -ex
```

Comment:
Where is this script used? Do you use it to manually test the image, or to showcase how to use the image?

Reply:
I use it to manually test the image. It uses the same arguments as [https://github.com/vmck/vmck/blob/master/contrib/build.py](vmck/contrib/build.py). Should I remove it?